docs/source/chatops/aliases.rst

```diff
@@ -71,7 +71,8 @@ To remove an action alias, use:
 
    st2 action-alias delete {ALIAS}
 
-
+After removing aliases, you may need to restart the ``st2chatops`` service, or the old commands may still show up
```

Comment:
Might want to also mention this should happen with changes, since the original reporter of this was making changes to the YAML definitions and that's where the duplicates were coming from.